src/main/java/org/folio/circulation/support/JsonPropertyWriter.java

```diff
@@ -54,7 +54,7 @@ public static void write(
 
   public static void write(JsonObject to, String propertyName, DateTime value) {
     if(value != null) {
-      write(to, propertyName, value.toString(ISODateTimeFormat.dateTime()));
+      write(to, propertyName, value.toString(ISODateTimeFormat.dateTime().withZoneUTC()));
```

Comment:
This change has been made to ensure that all date times are converted to UTC when written to JSON?

Reply:
Yes, when testing the time zone, it turned out that the `due date` value in the response to "checkout" is not the UTC. Date saves the applied zone.

Reply:
Ok, I think we may need to revisit the decision to do this conversion here. I would’ve thought that without this change, the code would use the timezone in the date time provided, which should be UTC

Reply:
I think this should be converted on the domain side. I'll revert the changes.

Reply:
Do you agree that I will revert the changes applied to `write()`?

Reply:
Yes, if the domain converts the relevant date and times to UTC, then please revert this chsnge to write

Reply:
done